vllm/_custom_ops.py

```diff
@@ -1135,6 +1135,53 @@ def scaled_fp4_experts_quant(
     return output, output_scales
 
 
+# Using the default value (240.0) from pytorch will cause accuracy
+# issue on dynamic quantization models. Here use 224.0 for rocm.
+FP8_DTYPE = current_platform.fp8_dtype()
+FP8_MAX = 224.0 if current_platform.is_rocm() else torch.finfo(FP8_DTYPE).max
+FP8_MIN = -224.0 if current_platform.is_rocm() else torch.finfo(FP8_DTYPE).min
+FP8_MIN_SCALING_FACTOR = 1.0 / (FP8_MAX * 512.0)
+
+
+@torch.compile()
```

Comment:
I think this will not fuse with other inductor ops because it's compiled separately. We should try to use a CustomOp subclass instead